### PR TITLE
Better Negative controls and their output

### PR DESCRIPTION
**Pull request by DarianHole**
## Added

- Adds if negative control issues are found to all output columns (easier to see info on IRIDA)

- Better way to automatically detect negative control names and put them automatically into the ncov-tools sample sheet
    - Thanks Phil for this

### CODE REVIEW

File: bin/qc.py [hunk 1]
```diff
@@ -187,6 +187,9 @@ def parse_ncov_tsv(file_in, sample, negative=False):
     if negative:
         new_columns = df.columns.values
         new_columns[0] = 'sample'
+        # Rename columns with negative_control to make them more informative when its ready in IRIDA
+        # for spot in range(1, len(new_columns)):
```
Comment (Takadonet) [inline bin/qc.py:191]: remove commented code.

File: extra_data/config.yaml [hunk 2]
```diff
@@ -55,11 +55,10 @@ assign_lineages: true
...
-# set the names of the negative controls
-#
-negative_control_samples: [ "PLACEHOLDER-negative_ctrl", "PLACEHOLDER-negative_ctrl-1", "PLACEHOLDER-negative_ctrl-2", "PLACEHOLDER-negative_ctrl-3", "negative_ctrl", "negative_ctrl-1", "negative_ctrl-2", "negative_ctrl-3" ]
-
 # minimum completeness threshold for inclusion to the SNP tree plot, if no entry
 # is provided the default is set to 0.75
 completeness_threshold: 0.75
+
+#
+# set the names of the negative controls
```
Comment (Takadonet) [inline extra_data/config.yaml:63]: add in text
"if control were found."

Comment (DarianHole) [reply extra_data/config.yaml]: added

File: bin/qc.py (hunk 1 above)
Comment (DarianHole) [reply bin/qc.py]: removed, forgot to do that earlier!

**Review by Takadonet — APPROVED** [review]

**Pull request merged**